src/assets/sass/modules/Achievements/_achievements.scss

```diff
@@ -0,0 +1,21 @@
+.achievements-block{
+    margin-top: 25px;
+}
```

Comment:
Indentation should be 2 spaces, no? This seems like 4.

Reply:
🆗